### PR TITLE
Support correct postal code events. 

### LINKED ISSUE #24: PostalCode does not use the correct events
The `PostalCode` component has a bunch of events defined on it but none of them match up with what is being fired from the underlying `StripeElement`. 

Most notably the "change" event is missing (instead there is `complete` but there doesn't appear to be any logic to actually emit that).

Also could you export the `StripeElement` as part of the `lib` build? It isn't part of the exports it is isn't even possible to create a custom `StripeElement` to work around the buggy `PostalCode`.

### PR DESCRIPTION
**Pull request by paulpdaniels**
@nkoehring @softbeehive 

Expose StripeElement for additional customization.

Fixes #24

### CODE REVIEW

**Comment by softbeehive** [comment]
@paulpdaniels, thank you for your contribution. It looks good, the only thing I worry in this particular case is backward compatibility.

File: src/PostalCode.vue [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
     :stripe='stripe'
     :value='value'
     :options='options'
     @blur='$emit("blur")'
     @focus='$emit("focus")'
     @empty='$emit("empty")'
-    @complete='$emit("complete")'
-    @brand='$emit("brand", $event)'
-    @error='$emit("error", $event)'
-    @value='$emit("value", $event)'
```
Comment (softbeehive) [inline src/PostalCode.vue]: @nkoehring, do you remember why does PostalCode.vue contain there listeners?
```
@complete='$emit("complete")'
@brand='$emit("brand", $event)'
@error='$emit("error", $event)'
@value='$emit("value", $event)'
```

I like the idea of replacing them with single `@change` but not sure if it is safe for those who will update the module

Comment (paulpdaniels) [reply src/PostalCode.vue]: @softbeehive this was just to bring them inline with the other components, which all use the same `change` event. Based on the commit message it seemed like they all used to have multiple events and this one was just missed?

Comment (softbeehive) [reply src/PostalCode.vue]: that's what it seems to me too, will test it and release if all good

Comment (paulpdaniels) [reply src/PostalCode.vue]: I remembered I could check git logs too. This looks like where they got out of sync. https://github.com/fromAtoB/vue-stripe-elements/commit/5218fbce5766741c58a9809ad8f1863619800e31#diff-25d902c24283ab8cfbac54dfa101ad31

**Pull request merged**